README.md

```diff
@@ -76,6 +78,7 @@ class Example extends React.Component {
               {pattern: /#(\w+)/,                 style: styles.hashTag},
             ]
           }
+          childrenProps={{ allowFontSize: false }}
```

Comment:
This should be labeled `allowFontScaling`, and remove the spaces between the braces for consistency with the other examples in the README